src/Misc/layoutbin/en-US/strings.json

```diff
@@ -249,10 +251,13 @@
   "RMArtifactFolderCreated": "Created artifact folder {0}",
   "RMArtifactMatchNotFound": "The build artifact '{0}' does not match any naming patterns, skipping download",
   "RMArtifactNameDirectoryNotFound": "Directory '{0}' does not exist. Falling back to parent directory: {1}",
+  "RMArtifactDownloadRequestCreationFailed": "Failed to create request to download artifact from URL: {0}",
```

Comment:
this is not in sorted order. there is a L0 test to validate sorted order of entries in this file. can you pl. run it & make sure it passes?